### PR TITLE
Fix value passed in the `--config` option to `fix-cs` Composer script

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
         }
     },
     "scripts": {
-        "fix-cs": "php bin/php-cs-fixer fix --config=.php_cs.dist"
+        "fix-cs": "php-cs-fixer fix --config=.php-cs-fixer.dist.php"
```
Comment (franmomu) [inline composer.json:87]:
```suggestion
        "fix-cs": "bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php"
```
To use the local `php-cs-fixer`

Comment (phansys) [reply composer.json:87]: It should use the local `php-cs-fixer`: https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands.

Comment (franmomu) [reply composer.json:87]: Didn't know about that 👍

**Review by franmomu — APPROVED** [review]

**Comment by franmomu** [comment]
Thanks @phansys!

**Pull request merged**